### DISCUSSION TITLE
Package groups requires case-sensitive package names

### DISCUSSION THREAD

**Discussion by bavedarnow** [renovatebot/renovate#9749] [Request Help]
When trying to group python packages, using the incorrect casing of the package name will not group them. I tried to find this info in the docs but didn't see it. 

does not work:
```
{
    "matchPackageNames": ["sqlalchemy", "sqlalchemy-utils"],
    "matchUpdateTypes": ["minor","patch"],
    "groupName": "SQLAlchemy",
}
```

works:
```
{
    "matchPackageNames": ["SQLAlchemy", "SQLAlchemy-Utils"],
    "matchUpdateTypes": ["minor","patch"],
    "groupName": "SQLAlchemy",
}
```

Is this the intended behavior? If so, I'd like to open a PR for the docs. Which section should this information be under?

https://docs.renovatebot.com/configuration-options/#packagerules
https://docs.renovatebot.com/configuration-options/#group

**Comment by rarkins** [comment]
Under matchPackageNames docs is best.

**Reply by rarkins** [reply]
It will match against the case of the dependency name as it's defined in the package file
